alacritty/src/window_context.rs

```diff
@@ -391,7 +392,11 @@ impl WindowContext {
 
         // Request immediate re-draw if visual bell animation is not finished yet.
         if !self.display.visual_bell.completed() {
-            self.display.window.request_redraw();
+            let window_id = Some(self.display.window.id());
+            // Winit on Windows discards `redraw_requested` calls from the `RedrawRequested` event,
+            // thus use a `Wakeup` which is delivered reliably and also sets the `dirty` flag.
+            let _ = event_proxy
+                .send_event(Event::new(EventType::Terminal(TerminalEvent::Wakeup), window_id));
```

Comment:
Using terminal events for windowing isn't an option imo.

Reply:
What's your suggestion then? To add one more event telling to refresh the display?